client/components/date-range/index.js

```diff
@@ -0,0 +1,665 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { noop, isNil, isNull, has } from 'lodash';
+import { DateUtils } from 'react-day-picker';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import { localize, moment } from 'i18n-calypso';
```

Comment:
Nit: 

`import { localize, moment } from 'i18n-calypso';`
and
`import Gridicon from 'gridicons';`
should live under `External dependencies`

Reply:
You keep having to remind me of this! Sorry. 

Fixed.

Reply:
NP! Easy to forget when there's more important stuff to fix :P